collect_app/src/main/res/drawable/baseline_rule_24.xml

```diff
@@ -0,0 +1,5 @@
+<vector android:autoMirrored="true" android:height="24dp"
+    android:tint="#000000" android:viewportHeight="24"
```

Comment:
In other icons if it's black we use `android:tint="@android:color/black"` so let's do the same here.